.travis.yml

```diff
@@ -22,7 +22,7 @@ before_install:
   - docker pull centos:7
 
 install:
-  - pip install tox ansible yamllint
+  - pip install tox ansible
```

Comment:
I think the reason for installing ansible was to have yamllint (plus something else that I already dropped I think). Is ansible still needed for anything when you add yamllint explicitly? Of course, you may need to change the `ansible --version` to `yamllint --version` below.

Reply:
I left it there purely for the `ansible --version` to work as I wasn't sure if that was still needed (maybe it is a very important debugging artifact).

Reply:
I have a cleaner version locally now.